### PR TITLE
Implemented the PKHTB and PKHBT instructions with tests.

### PR DESCRIPTION
**Pull request by Subv**

### CODE REVIEW

File: tests/arm/fuzz_arm.cpp [hunk 1]
```diff
@@ -904,3 +904,23 @@ TEST_CASE("VFP: VPUSH, VPOP", "[JitX64][vfp]") {
...
+    auto is_pkh_valid = [](u32 inst) -> bool {
+        // R15 as Rd, Rn, or Rm is UNPREDICTABLE
+        return Bits<16, 19>(inst) != 0b1111 &&
+               Bits<12, 15>(inst) != 0b1111 &&
+               Bits<0, 3>(inst) != 0b1111;
+    };
+
+    const std::array<InstructionGenerator, 2> instructions = {{
+        InstructionGenerator("cccc01101000nnnnddddvvvvv001mmmm", is_pkh_valid), // PKHBT
+        InstructionGenerator("cccc01101000nnnnddddvvvvv101mmmm", is_pkh_valid),   // PKHTB
```
Comment (wwylele) [inline tests/arm/fuzz_arm.cpp]: nit: space

**Comment by Subv** [comment]
Addressed.

**Comment by MerryMage** [comment]
LGTM

**Comment by Subv** [comment]
Done

**Pull request merged**